tools/jvm/src/main/scala/org/scalajs/core/tools/io/package.scala

```diff
@@ -0,0 +1,56 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___      __ ____  Scala.js tools             **
+**    / __/ __// _ | / /  / _ | __ / // __/  (c) 2013-2014, LAMP/EPFL   **
+**  __\ \/ /__/ __ |/ /__/ __ |/_// /_\ \    http://scala-js.org/       **
+** /____/\___/_/ |_/____/_/ | |__/ /____/                               **
+**                          |/____/                                     **
+\*                                                                      */
+
+
+package org.scalajs.core.tools
+
+import org.scalajs.core.tools.io.IRFileCache.IRContainer
+
+import java.io.File
+
+package object io {
+
+  implicit class RichIRContainerCompanion(
+      val companion: IRContainer.type) extends AnyVal {
```

Comment:
I'm wondering whether a trait mixed in `IRContainer` wouldn't be more appropriate. The trait would be defined separately in js/ and in jvm/ and have self-type of `IRContainer.type`.

This has two advantages: it doesn't require to import `io._`, and it allows to access those methods if one does `import IRContainer._`.